CRITICAL - 2024-08-27 21:46:51 --> InvalidArgumentException: You have not selected a database type to connect to.
[Method: POST, Route: registerUser]
in SYSTEMPATH\Database\Database.php on line 54.
 1 SYSTEMPATH\Database\Config.php(84): CodeIgniter\Database\Database->load([...], 'default')
 2 SYSTEMPATH\Model.php(155): CodeIgniter\Database\Config::connect('default')
 3 APPPATH\Controllers\AuthController.php(15): CodeIgniter\Model->__construct()
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-27 21:55:21 --> InvalidArgumentException: You have not selected a database type to connect to.
[Method: POST, Route: registerUser]
in SYSTEMPATH\Database\Database.php on line 54.
 1 SYSTEMPATH\Database\Config.php(84): CodeIgniter\Database\Database->load([...], 'default')
 2 SYSTEMPATH\Model.php(155): CodeIgniter\Database\Config::connect('default')
 3 APPPATH\Controllers\AuthController.php(15): CodeIgniter\Model->__construct()
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-27 21:55:27 --> InvalidArgumentException: You have not selected a database type to connect to.
[Method: POST, Route: registerUser]
in SYSTEMPATH\Database\Database.php on line 54.
 1 SYSTEMPATH\Database\Config.php(84): CodeIgniter\Database\Database->load([...], 'default')
 2 SYSTEMPATH\Model.php(155): CodeIgniter\Database\Config::connect('default')
 3 APPPATH\Controllers\AuthController.php(15): CodeIgniter\Model->__construct()
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-27 21:57:28 --> InvalidArgumentException: You have not selected a database type to connect to.
[Method: POST, Route: registerUser]
in SYSTEMPATH\Database\Database.php on line 54.
 1 SYSTEMPATH\Database\Config.php(84): CodeIgniter\Database\Database->load([...], 'default')
 2 SYSTEMPATH\Model.php(155): CodeIgniter\Database\Config::connect('default')
 3 APPPATH\Controllers\AuthController.php(15): CodeIgniter\Model->__construct()
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-27 21:57:37 --> InvalidArgumentException: You have not selected a database type to connect to.
[Method: POST, Route: registerUser]
in SYSTEMPATH\Database\Database.php on line 54.
 1 SYSTEMPATH\Database\Config.php(84): CodeIgniter\Database\Database->load([...], 'default')
 2 SYSTEMPATH\Model.php(155): CodeIgniter\Database\Config::connect('default')
 3 APPPATH\Controllers\AuthController.php(15): CodeIgniter\Model->__construct()
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-27 21:58:48 --> InvalidArgumentException: You have not selected a database type to connect to.
[Method: POST, Route: registerUser]
in SYSTEMPATH\Database\Database.php on line 54.
 1 SYSTEMPATH\Database\Config.php(84): CodeIgniter\Database\Database->load([...], 'default')
 2 SYSTEMPATH\Model.php(155): CodeIgniter\Database\Config::connect('default')
 3 APPPATH\Controllers\AuthController.php(15): CodeIgniter\Model->__construct()
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-27 21:59:30 --> Error connecting to the database: mysqli_sql_exception: Invalid character set was provided in C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php:210
Stack trace:
#0 C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php(210): mysqli->set_charset('')
#1 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\prueba\eskate\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\prueba\eskate\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\prueba\eskate\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\prueba\eskate\app\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
#10 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\prueba\eskate\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\prueba\eskate\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\prueba\eskate\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Invalid character set was provided in C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\prueba\eskate\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\prueba\eskate\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\prueba\eskate\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#6 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#7 C:\xampp\htdocs\prueba\eskate\app\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save(Array)
#8 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
#9 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\prueba\eskate\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\prueba\eskate\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\prueba\eskate\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-08-27 21:59:30 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [mysqli]: Invalid character set was provided
[Method: POST, Route: registerUser]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-27 22:00:00 --> InvalidArgumentException: You have not selected a database type to connect to.
[Method: POST, Route: registerUser]
in SYSTEMPATH\Database\Database.php on line 54.
 1 SYSTEMPATH\Database\Config.php(84): CodeIgniter\Database\Database->load([...], 'default')
 2 SYSTEMPATH\Model.php(155): CodeIgniter\Database\Config::connect('default')
 3 APPPATH\Controllers\AuthController.php(15): CodeIgniter\Model->__construct()
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-27 22:00:56 --> Error connecting to the database: mysqli_sql_exception: Invalid character set was provided in C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php:210
Stack trace:
#0 C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php(210): mysqli->set_charset('')
#1 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\prueba\eskate\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\prueba\eskate\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\prueba\eskate\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\prueba\eskate\app\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
#10 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\prueba\eskate\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\prueba\eskate\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\prueba\eskate\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Invalid character set was provided in C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\prueba\eskate\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\prueba\eskate\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\prueba\eskate\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#6 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#7 C:\xampp\htdocs\prueba\eskate\app\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save(Array)
#8 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
#9 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\prueba\eskate\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\prueba\eskate\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\prueba\eskate\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-08-27 22:00:56 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [mysqli]: Invalid character set was provided
[Method: POST, Route: registerUser]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-27 22:00:59 --> Error connecting to the database: mysqli_sql_exception: Invalid character set was provided in C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php:210
Stack trace:
#0 C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php(210): mysqli->set_charset('')
#1 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\prueba\eskate\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\prueba\eskate\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\prueba\eskate\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\prueba\eskate\app\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
#10 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\prueba\eskate\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\prueba\eskate\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\prueba\eskate\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Invalid character set was provided in C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\prueba\eskate\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\prueba\eskate\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\prueba\eskate\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#6 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#7 C:\xampp\htdocs\prueba\eskate\app\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save(Array)
#8 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
#9 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\prueba\eskate\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\prueba\eskate\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\prueba\eskate\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-08-27 22:00:59 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [mysqli]: Invalid character set was provided
[Method: POST, Route: registerUser]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-27 22:01:31 --> Error connecting to the database: mysqli_sql_exception: Invalid character set was provided in C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php:210
Stack trace:
#0 C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php(210): mysqli->set_charset('')
#1 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\prueba\eskate\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\prueba\eskate\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\prueba\eskate\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\prueba\eskate\app\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
#10 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\prueba\eskate\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\prueba\eskate\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\prueba\eskate\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Invalid character set was provided in C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\prueba\eskate\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\prueba\eskate\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\prueba\eskate\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#6 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#7 C:\xampp\htdocs\prueba\eskate\app\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save(Array)
#8 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
#9 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\prueba\eskate\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\prueba\eskate\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\prueba\eskate\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-08-27 22:01:31 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [mysqli]: Invalid character set was provided
[Method: POST, Route: registerUser]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-27 22:02:58 --> Error connecting to the database: mysqli_sql_exception: Invalid character set was provided in C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php:210
Stack trace:
#0 C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php(210): mysqli->set_charset('')
#1 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\prueba\eskate\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\prueba\eskate\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\prueba\eskate\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\prueba\eskate\app\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
#10 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\prueba\eskate\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\prueba\eskate\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\prueba\eskate\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Invalid character set was provided in C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\prueba\eskate\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\prueba\eskate\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\prueba\eskate\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#6 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#7 C:\xampp\htdocs\prueba\eskate\app\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save(Array)
#8 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
#9 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\prueba\eskate\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\prueba\eskate\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\prueba\eskate\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-08-27 22:02:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [mysqli]: Invalid character set was provided
[Method: POST, Route: registerUser]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-27 22:33:40 --> ErrorException: require_once(/views/welcome.php): Failed to open stream: No such file or directory
[Method: GET, Route: welcome]
in APPPATH\Controllers\AuthController.php on line 64.
 1 APPPATH\Controllers\AuthController.php(64): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require_once(/views/welcome.php): Failed to open stream: No such file or directory', 'C:\\xampp\\htdocs\\prueba\\eskate\\app\\Controllers\\AuthController.php', 64)
 2 APPPATH\Controllers\AuthController.php(64): require_once()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->welcome()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-27 22:33:59 --> ErrorException: require_once(./views/welcome.php): Failed to open stream: No such file or directory
[Method: GET, Route: welcome]
in APPPATH\Controllers\AuthController.php on line 64.
 1 APPPATH\Controllers\AuthController.php(64): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require_once(./views/welcome.php): Failed to open stream: No such file or directory', 'C:\\xampp\\htdocs\\prueba\\eskate\\app\\Controllers\\AuthController.php', 64)
 2 APPPATH\Controllers\AuthController.php(64): require_once()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->welcome()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-27 22:34:12 --> ErrorException: require_once(./Views/welcome.php): Failed to open stream: No such file or directory
[Method: GET, Route: welcome]
in APPPATH\Controllers\AuthController.php on line 64.
 1 APPPATH\Controllers\AuthController.php(64): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require_once(./Views/welcome.php): Failed to open stream: No such file or directory', 'C:\\xampp\\htdocs\\prueba\\eskate\\app\\Controllers\\AuthController.php', 64)
 2 APPPATH\Controllers\AuthController.php(64): require_once()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->welcome()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-27 23:06:30 --> ParseError: syntax error, unexpected token "/"
[Method: GET, Route: login]
in APPPATH\Views\login.php on line 71.
 1 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1169): CodeIgniter\View\View->render('login', [], true)
 3 APPPATH\Controllers\AuthController.php(30): view('login')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-27 23:06:31 --> ParseError: syntax error, unexpected token "/"
[Method: GET, Route: login]
in APPPATH\Views\login.php on line 71.
 1 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1169): CodeIgniter\View\View->render('login', [], true)
 3 APPPATH\Controllers\AuthController.php(30): view('login')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-27 23:21:18 --> mysqli_sql_exception: Column 'username' cannot be null in C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\prueba\eskate\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\prueba\eskate\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\prueba\eskate\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\prueba\eskate\app\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
#10 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\prueba\eskate\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\prueba\eskate\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\prueba\eskate\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-08-27 23:21:18 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'username' cannot be null
[Method: POST, Route: registerUser]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-27 23:21:18 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'username' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`) VALUES (NULL, \'1234@gmail.com\', \'$2y$10$9lyafjGiBqG6Sp/CSh8.WuEBPHDRG2/tfjS0QFI8esYcWLq4rzOXi\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`) VALUES (NULL, \'1234@gmail.com\', \'$2y$10$9lyafjGiBqG6Sp/CSh8.WuEBPHDRG2/tfjS0QFI8esYcWLq4rzOXi\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-27 23:21:18 --> [Caused by] mysqli_sql_exception: Column 'username' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (NULL, \'1234@gmail.com\', \'$2y$10$9lyafjGiBqG6Sp/CSh8.WuEBPHDRG2/tfjS0QFI8esYcWLq4rzOXi\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`) VALUES (NULL, \'1234@gmail.com\', \'$2y$10$9lyafjGiBqG6Sp/CSh8.WuEBPHDRG2/tfjS0QFI8esYcWLq4rzOXi\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`) VALUES (NULL, \'1234@gmail.com\', \'$2y$10$9lyafjGiBqG6Sp/CSh8.WuEBPHDRG2/tfjS0QFI8esYcWLq4rzOXi\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-27 23:22:16 --> mysqli_sql_exception: Column 'username' cannot be null in C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\prueba\eskate\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\prueba\eskate\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\prueba\eskate\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\prueba\eskate\app\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
#10 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\prueba\eskate\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\prueba\eskate\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\prueba\eskate\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-08-27 23:22:16 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'username' cannot be null
[Method: POST, Route: registerUser]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-27 23:22:16 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'username' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`) VALUES (NULL, \'1234@gmail.com\', \'$2y$10$yL7FtigkdplxQBMiZeuleO0jkOj7wk2oqjyzoSsCSOiQN98un1dKO\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`) VALUES (NULL, \'1234@gmail.com\', \'$2y$10$yL7FtigkdplxQBMiZeuleO0jkOj7wk2oqjyzoSsCSOiQN98un1dKO\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-27 23:22:16 --> [Caused by] mysqli_sql_exception: Column 'username' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (NULL, \'1234@gmail.com\', \'$2y$10$yL7FtigkdplxQBMiZeuleO0jkOj7wk2oqjyzoSsCSOiQN98un1dKO\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`) VALUES (NULL, \'1234@gmail.com\', \'$2y$10$yL7FtigkdplxQBMiZeuleO0jkOj7wk2oqjyzoSsCSOiQN98un1dKO\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`) VALUES (NULL, \'1234@gmail.com\', \'$2y$10$yL7FtigkdplxQBMiZeuleO0jkOj7wk2oqjyzoSsCSOiQN98un1dKO\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-27 23:22:29 --> mysqli_sql_exception: Column 'username' cannot be null in C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\prueba\eskate\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\prueba\eskate\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\prueba\eskate\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\prueba\eskate\app\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
#10 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\prueba\eskate\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\prueba\eskate\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\prueba\eskate\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-08-27 23:22:29 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'username' cannot be null
[Method: POST, Route: registerUser]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-27 23:22:29 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'username' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`) VALUES (NULL, \'1234@gmail.com\', \'$2y$10$9YlG3uRBp/fJiddsM/5nAutkIwXGWQcTiVgsk37FC0eEvH3JbDRqq\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`) VALUES (NULL, \'1234@gmail.com\', \'$2y$10$9YlG3uRBp/fJiddsM/5nAutkIwXGWQcTiVgsk37FC0eEvH3JbDRqq\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-27 23:22:29 --> [Caused by] mysqli_sql_exception: Column 'username' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (NULL, \'1234@gmail.com\', \'$2y$10$9YlG3uRBp/fJiddsM/5nAutkIwXGWQcTiVgsk37FC0eEvH3JbDRqq\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`) VALUES (NULL, \'1234@gmail.com\', \'$2y$10$9YlG3uRBp/fJiddsM/5nAutkIwXGWQcTiVgsk37FC0eEvH3JbDRqq\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`) VALUES (NULL, \'1234@gmail.com\', \'$2y$10$9YlG3uRBp/fJiddsM/5nAutkIwXGWQcTiVgsk37FC0eEvH3JbDRqq\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-27 23:23:10 --> mysqli_sql_exception: Column 'username' cannot be null in C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\prueba\eskate\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\prueba\eskate\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\prueba\eskate\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\prueba\eskate\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\prueba\eskate\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\prueba\eskate\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\prueba\eskate\app\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
#10 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#11 C:\xampp\htdocs\prueba\eskate\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\prueba\eskate\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\prueba\eskate\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\prueba\eskate\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-08-27 23:23:10 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'username' cannot be null
[Method: POST, Route: registerUser]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-27 23:23:10 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'username' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`) VALUES (NULL, NULL, \'$2y$10$79BfY1lc5KfsAMdEip7qOO8EDtFqbNueM6HLMwaz4zOc1VrWsd7CS\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`) VALUES (NULL, NULL, \'$2y$10$79BfY1lc5KfsAMdEip7qOO8EDtFqbNueM6HLMwaz4zOc1VrWsd7CS\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-27 23:23:10 --> [Caused by] mysqli_sql_exception: Column 'username' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (NULL, NULL, \'$2y$10$79BfY1lc5KfsAMdEip7qOO8EDtFqbNueM6HLMwaz4zOc1VrWsd7CS\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`username`, `email`, `password`) VALUES (NULL, NULL, \'$2y$10$79BfY1lc5KfsAMdEip7qOO8EDtFqbNueM6HLMwaz4zOc1VrWsd7CS\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`username`, `email`, `password`) VALUES (NULL, NULL, \'$2y$10$79BfY1lc5KfsAMdEip7qOO8EDtFqbNueM6HLMwaz4zOc1VrWsd7CS\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`username`, `email`, `password`) VALUES (:username:, :email:, :password:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\AuthController.php(23): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->registerUser()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
